test/controllers/EntryController.test.ts

```diff
@@ -0,0 +1,24 @@
+import 'reflect-metadata'
+import '../../src/controllers/EntryController'
+import { createApplication } from '../ApplicationFactory';
+import * as request from 'supertest'
+import { RedisService } from '../../src/services/RedisService';
+import { createSubstituteOf } from '../SubstituteFactory';
+import { MOVED_TEMPORARILY  } from 'http-status-codes';
+import { ENTRY_URI, PENALTY_DETAILS_PREFIX} from '../../src/utils/Paths';
+import { expect } from 'chai';
+
+describe('EntryController', () => {
+
+    describe('GET request', () => {
+        it('should return 302 when trying to access the entry page', async () => {
+            const app = createApplication(container => {
```

Comment:
To be fair you do not need RedisService. Do `createApplication()` and you are all set.

Reply:
Error: No matching bindings found for serviceIdentifier: RedisService

Reply:
Hmm, I bet it works fine when you run one test only.